core/src/main/java/io/questdb/griffin/ExpressionParser.java

```diff
@@ -849,6 +853,9 @@ void parseExpr(GenericLexer lexer, ExpressionParserListener listener, SqlParserC
                                     }
                                 }
 
+                                // there is one case for valid dangling expression - when we create alias for column (like 'value' 'x' equivalent to 'value' as 'x')
+                                // this helper works for easy cases leaving unparsed last token and give a chance for caller to analyze aliasing
+                                // although for complex cases it will not work (like 'a' || 'b' 'x' will not be parsed as 'a' || 'b' as 'x' without explicit braces)
```

Comment:
nit: like is a SQL keyword
```suggestion
                                // although for complex cases it will not work (`'a' || 'b' 'x'` will not be parsed as `'a' || 'b' as 'x'` without explicit braces)
```